### DISCUSSION TITLE
useQuery export error

### DISCUSSION THREAD

**Discussion by sanpanthera** [TanStack/query#845] [General] [answered]
I am getting below error while building the app.

![image](https://user-images.githubusercontent.com/24603729/89772303-6bf66380-db1f-11ea-8e8f-07ae23bec506.png)

Please guide me to resolve this issue.

**Comment by sanpanthera** [comment]
I found the root cause.. In the react-query's package.json module key is pointing to  mjs file. If I point to any other .js file within dist folder it works...
![image](https://user-images.githubusercontent.com/24603729/90249731-9bc3a500-de58-11ea-85df-7386eab5745c.png)

Can someone help on this?

**Comment by jackmellis** [comment, marked as answer]
Does your bundler config know how to read `.mjs` files? For example in webpack you have you to specify `extensions: [ '.js', '.jsx', '.mjs' ]` etc.
